lib/grammars.coffee

```diff
@@ -868,3 +868,12 @@ module.exports =
         uri = 'file://' + context.filepath
         shell.openExternal(uri)
         ['HTML file opened at:', uri]
+  
+  MQL5:
+    if GrammarUtils.OperatingSystem.isDarwin()
```

Comment:
Judging by the fact that you're using `cmd.exe` and a `Program Files` path, do you mean for this conditional to check if it's a Windows machine?